specs/RuntimeSpec/ApplyingChunk.md

```diff
@@ -0,0 +1,37 @@
+# Processing chunk
```

Comment:
"Processing chunk" sounds like sometime happens in chain, not runtime. Maybe renaming it to "Apply chunks" would be better?

Reply:
```suggestion
# Applying chunk
```